From 93f12a439d613e17182dcab28f0584b08f9577cc Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Fri, 8 Jan 2016 18:51:07 +0800
Subject: [PATCH] MLK-12190 ARM: dts: imx7d-sdb-revb: change pad setting to
 remove stripe

Change GPIO pad setting to control EPDC/ENET signals on REV B board from 0x59
to default one 0x14 in order to remove stripe when do EPDC unit test.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
(cherry picked from commit 25a76f98eb3c556b4c2676e84cd3e205f7a93ad2)
---
 arch/arm/boot/dts/imx7d-sdb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index a206341..56a0511 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -616,7 +616,7 @@
 
 		pinctrl_enet2_epdc0_en: enet2_epdc0_grp {
 			fsl,pins = <
-				MX7D_PAD_GPIO1_IO04__GPIO1_IO4		0x59
+				MX7D_PAD_GPIO1_IO04__GPIO1_IO4		0x80000000
 			>;
 		};
 
-- 
1.8.0

